src/test_workflow/benchmark_test/benchmark_test_cluster.py

```diff
@@ -132,8 +134,12 @@ def setup_cdk_params(self, config: dict) -> dict:
             artifact_url = self.manifest.build.location if isinstance(self.manifest, BundleManifest) else \
                 f"https://artifacts.opensearch.org/snapshots/core/opensearch/{self.manifest.build.version}/opensearch-min-" \
                 f"{self.manifest.build.version}-linux-{self.manifest.build.architecture}-latest.tar.gz"
+            if not self.args.insecure and semver.compare(self.manifest.build.version, '2.12.0') != -1:
+                need_strong_password = True
```

Comment:
@rishabh6788 looks to me like this only checks for 2.12.0 which is good, but anything above 2.12.0 need this `need_strong_password` right ?

Reply:
this checks if the passed version is equal to or grater then 2.12.0, if it is then strong password will be used. If not then this is ignored. Anything after 2.12.0 would anyway be using strong password.